{"uuid": "cb737aa4-9389-4a11-81b6-8a0ff5244b91", "children": ["860d608f-0672-4062-8e54-d0d0e87ebd51"], "befores": [{"name": "app", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: Process unexpectedly closed with status 1\n\n", "trace": "  File \"/home/sash/PycharmProjects/training/env/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/sash/PycharmProjects/training/env/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/sash/PycharmProjects/training/env/lib/python3.8/site-packages/_pytest/fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/sash/PycharmProjects/training/conftest.py\", line 35, in app\n    fixture = Application(browser=browser, baseUrl=target[\"web\"][\"baseUrl\"], photoPath=root_path)\n  File \"/home/sash/PycharmProjects/training/fixture/application.py\", line 17, in __init__\n    self.wd = webdriver.Firefox()\n  File \"/home/sash/PycharmProjects/training/env/lib/python3.8/site-packages/selenium/webdriver/firefox/webdriver.py\", line 170, in __init__\n    RemoteWebDriver.__init__(\n  File \"/home/sash/PycharmProjects/training/env/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 157, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/home/sash/PycharmProjects/training/env/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 252, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"/home/sash/PycharmProjects/training/env/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"/home/sash/PycharmProjects/training/env/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1632890777230, "stop": 1632890777339}], "start": 1632890777230, "stop": 1632890777403}